src/metabase/public_settings.clj

```diff
@@ -224,8 +224,7 @@
 (defsetting enable-query-caching
   (deferred-tru "Enabling caching will save the results of queries that take a long time to run.")
   :type    :boolean
-  :default false
-  :visibility :settings-manager)
+  :default false)
```

Comment:
Oh, thanks for catching that. I didn't realize this setting was read outside of the admin panel (and obviously Cypress didn't catch it).

However I think this should actually be `:visibility :authenticated`. Otherwise only admins will see this in the info sidebar, since `:admin` is the default visibility level.

```suggestion
  :default    :false
  :visibility :authenticated)
```

Reply:
great catch, thanks!